### PR TITLE
Pil 817 - Address accessability issues including WCAG issues.

### PR DESCRIPTION
**Pull request by MarjanMubarok**
PR for the accessabiilty issues in PIL-817.

### CODE REVIEW

File: app/views/CheckYourAnswersView.scala.html [hunk 1]
```diff
@@ -15,47 +15,57 @@
...
-    <h2 class="govuk-heading-m">@messages("contactCheckYourAnswers.second")</h2>
+        @heading(messages("contactCheckYourAnswers.address"), "govuk-heading-m", "h2")
+        @govukSummaryList(address)
 
-    @govukSummaryList(listSecondary)
-    <h2 class="govuk-heading-m">@messages("contactCheckYourAnswers.address")</h2>
-    @govukSummaryList(address)
+        @heading(messages("checkYourAnswers.submit.registration"), "govuk-heading-m", "h2")
+        @paragraphBody(messages("checkYourAnswers.submit.p"), "govuk-body")
+        <br>
```
Comment (jassalrichy) [inline app/views/CheckYourAnswersView.scala.html:64]: remove <\br\> and use class on govukButton to add space between elements

File: test/forms/mappings/DateMappingsSpec.scala [hunk 2]
```diff
@@ -50,6 +50,12 @@ class DateMappingsSpec extends AnyFreeSpec with Matchers with ScalaCheckProperty
   )
 
   val invalidField: Gen[String] = Gen.alphaStr.suchThat(_.nonEmpty)
+  def numberStringInRangeGen(min: Int, max: Int): Gen[String] =
+    Gen.chooseNum(min, max).map(_.toString)
+//  val invalidMonth: Gen[Int] = intsAboveValue(12)
```
Comment (jassalrichy) [inline test/forms/mappings/DateMappingsSpec.scala:55]: remove commented out line if not required

Comment (MarjanMubarok) [reply test/forms/mappings/DateMappingsSpec.scala:55]: :blush: Going to remove the generators as well looks like there is a test case which covers it already without needing to use ScalaCheck generators.

File: app/views/CheckYourAnswersView.scala.html (hunk 1 above)
Comment (MarjanMubarok) [reply app/views/CheckYourAnswersView.scala.html]: <br> has been removed. The GDS button component is already being used.

**Review by aashik-varahan — DISMISSED** [review]

**Review by javed-iqbal1 — APPROVED** [review]

**Review by aashik-varahan — DISMISSED** [review]

**Review by aashik-varahan — APPROVED** [review]

**Pull request merged**